app/assets/javascripts/discourse/tests/acceptance/user-private-messages-test.js

```diff
@@ -971,52 +970,3 @@ acceptance(
     });
   }
 );
-
-acceptance(
-  "User Private Messages - user with uppercase username",
-  function (needs) {
-    needs.user({
-      groups: [{ id: 14, name: "awesome_group", has_messages: true }],
-    });
-
-    needs.pretender((server, helper) => {
-      const response = cloneJSON(userFixtures["/u/charlie.json"]);
-      response.user.username = "chArLIe";
-      server.get("/u/charlie.json", () => helper.response(response));
-
-      server.get(
-        "/topics/private-messages-group/:username/:group_name.json",
-        () => {
-          return helper.response({
-            topic_list: {
-              topics: [
-                { id: 1, posters: [] },
-                { id: 2, posters: [] },
-              ],
-            },
-          });
-        }
-      );
-    });
-
-    test("viewing inbox", async function (assert) {
-      await visit("/u/charlie/messages");
-
-      assert.strictEqual(
-        query(".user-nav-messages-dropdown .selected-name").textContent.trim(),
-        "Inbox",
-        "menu defaults to Inbox"
-      );
-    });
-
-    test("viewing group inbox", async function (assert) {
-      await visit("/u/charlie/messages/group/awesome_group");
-
-      assert.strictEqual(
-        query(".user-nav-messages-dropdown .selected-name").textContent.trim(),
-        "awesome_group",
-        "dropdown menu displays the right group name"
-      );
-    });
-  }
-);
```

Comment:
Changing to a system test because it is a nightmare to get the fixtures right here.